Dockerfile

```diff
@@ -4,8 +4,14 @@ MAINTAINER Marek Goldmann <mgoldman@redhat.com>
 # User root user to install software
 USER root
 
-# Install necessary packages
-RUN yum -y install java-1.8.0-openjdk-devel && yum clean all
+# Install OpenJDK 11
+WORKDIR /opt
+RUN yum install wget -y \
+	&& wget https://download.java.net/java/ga/jdk11/openjdk-11_linux-x64_bin.tar.gz \
```

Comment:
Why not using OpenJDK provided in CentOS?

Reply:
Take a look at the `java-11-openjdk-devel` package. It's new in CentOS 7.6.

Reply:
You're right, thank you :)